enterprise/coderd/provisionerdaemons.go

```diff
@@ -196,6 +198,12 @@ func (api *API) provisionerDaemonServe(rw http.ResponseWriter, r *http.Request)
 		Tags:         tags,
 	})
 	if err != nil {
+		api.Logger.Error(ctx, "write provisioner daemon",
```

Comment:
Suggestion, consider `logger = api.Logger.With(...)` for the common tags and updating so use of logger. This way the useful context will be available elsewhere too.